.circleci/config.yml

```diff
@@ -47,6 +47,24 @@ jobs:
             else
               echo "Redis success: Redis contains $redisKeys keys."
             fi
+      - run:
```

Comment:
Moved testing of Nginx configuration before deployment command. These tests do not modify created images in any way, so deployment can still use the same images built within previous steps.